### ISSUE TITLE
Improve plugin interface

### ISSUE THREAD

**Issue by unional** [unional/komondor#55]
It is getting a bit messy for my taste.

```ts
interface KomondorPluginUtil {
  registerXXX,
  getSpy(...),
  getStub(...),
  getReturnSpy(...),
  getReturnStub(...),
  log...,
}

export function activate(komondor: KomondorPluginUtil) {
  komondor.registerSpy(...)
  komondor.registerStub(...)
  komondor.registerReturnSpy(...)
  komondor.registerReturnStub(...)
}
```

**Comment by unional** [comment]
And expose the plugin system.

**Comment by unional** [comment]
fixed by #72
